lib/src/modules/conditionals/if.dart

```diff
@@ -82,10 +88,14 @@ class If extends Conditional {
 
   /// If [condition] is high, then [then] executes, otherwise [orElse] is
   /// executed.
-  If(Logic condition, {List<Conditional>? then, List<Conditional>? orElse})
+  If(Logic condition,
+      {List<Conditional>? then,
+      List<Conditional>? orElse,
+      String? ifLabel,
```

Comment:
similar to prior comments: what if we just accepted one `label`/`name` and named the blocks `if_$name` and `else_$name` or something automatically?  or do you think there's enough demand likely for people to customize the if and else block names separately?  and if so, maybe it's ok to expect them to use the more verbose version of `If.block`?